charts/csi-powermax/charts/csireverseproxy/conf/config.yaml

```diff
@@ -35,4 +35,4 @@ config:
         maxOutStandingRead: {{ $value.limits.maxOutStandingRead | default 0 }}
         maxOutStandingWrite: {{ $value.limits.maxOutStandingWrite | default 0 }}
       {{- end }}
-    {{- end }}
+    {{- end }}
```

Comment:
add new line

Reply:
When I added new line the linter check got failed. So, removed the extra line.